packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1165,7 +1165,8 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
                     },
                     this.runtimeOptions.summaryOptions.summarizerOptions,
                 );
-                this.summaryManager.on("summarizerWarning", this.raiseContainerWarning);
+                this.summaryManager.on("summarizerWarning", (warning: ContainerWarning) =>
+                    this.mc.logger.sendTelemetryEvent({ eventName: "summarizerWarning" }, warning));
```

Comment:
I'd remove "summarizerWarning" concept altogether:
1. SummarizerException case - we can remove warning altogether, as it's already logged to telemetry (as an error).
2. delayBeforeCreatingSummarizer() - I'd probably not even log anything, as "CreatingSummarizer" event below has all the same info (if it does not, I'd add it to its payload)
3. "summarizingError" - I'd log anything coming this pipe with category = "error", not non-error event.

Reply:
I'd prefer to open this telemetry change as a separate issue for consideration.

Reply:
Follow-up work item for `summarizerWarning`: https://github.com/microsoft/FluidFramework/issues/8903